ec2/variables.tf

```diff
@@ -49,3 +49,12 @@ variable "attach_policies" {
   type        = map(string)
   default     = {}
 }
+
+variable "role_name" {
+  default     = ""
+  description = "Allows the role for the instance to be overridden with an existing one."
```

Comment:
Not sure that description make sense? Did you mean: "Allows the role for the instance to be overridden with a different one."

Reply:
I meant that the role already exists but yours makes more sense. I've changed it.